### PR TITLE
Add option to resend fax

### PR DESCRIPTION
**Pull request by R0MANDEV**
#### Type Of Change 
- [x] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: web/portal/client/src/translations/es.json [hunk 1]
```diff
@@ -456,6 +456,8 @@
   "Registered until: {{date}}": "Registrado hasta: {{date}}",
   "Regular Expression": "Expresión regular",
   "Regular NAT detected": "NAT normal detectado",
+  "Resend faxes": "Reenviar faxs",
```
Comment (mmadariaga) [inline web/portal/client/src/translations/es.json:459]: faxs

File: web/portal/client/src/translations/es.json [hunk 2]
```diff
@@ -456,6 +456,8 @@
...
+  "Resend faxes?": "Reenviar faxs?",
```
Comment (mmadariaga) [inline web/portal/client/src/translations/es.json:460]: faxs

File: web/portal/client/src/translations/es.json [hunk 3]
```diff
@@ -583,6 +585,7 @@
   "Wholesale": "Wholesale",
   "Work Phone": "Teléfono Trabajo",
   "Yes": "Sí",
+  "You are about to resend this faxes": "Está a punto de reenviar este faxs.",
```
Comment (mmadariaga) [inline web/portal/client/src/translations/es.json:588]: faxs

File: web/portal/client/src/entities/FaxesOut/Action/index.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { CustomActionsType } from '@irontec/ivoz-ui/entities/EntityInterface';
+
+import Resend from './Resend';
+
+const customAction: CustomActionsType = {
+  Regenerate: {
```
Comment (mmadariaga) [inline web/portal/client/src/entities/FaxesOut/Action/index.ts:6]:
```suggestion
  Resend: {
```

File: web/portal/client/src/entities/FaxesOut/Action/Resend.tsx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+      setOpen(false);
+      reload();
+    });
+  };
+
+  return (
+    <>
+      {variant === 'text' && (
+        <MoreMenuItem onClick={handleClickOpen}>
+          {_('Resend faxes')}
```
Comment (mmadariaga) [inline web/portal/client/src/entities/FaxesOut/Action/Resend.tsx:63]:
```suggestion
          {_('Resend fax')}
```
it's singular

File: web/portal/client/src/entities/FaxesOut/Action/Resend.tsx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+        </MoreMenuItem>
+      )}
+      {variant === 'icon' && (
+        <Tooltip
+          title={_('Resend faxes')}
```
Comment (mmadariaga) [inline web/portal/client/src/entities/FaxesOut/Action/Resend.tsx:68]:
```suggestion
          title={_('Resend fax')}
```

File: web/portal/client/src/entities/FaxesOut/Action/Resend.tsx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+        </Tooltip>
+      )}
+
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby='alert-dialog-title'
+        aria-describedby='alert-dialog-description'
+      >
+        <DialogTitle id='alert-dialog-title'>{_('Resend faxes?')}</DialogTitle>
```
Comment (mmadariaga) [inline web/portal/client/src/entities/FaxesOut/Action/Resend.tsx:86]:
```suggestion
        <DialogTitle id='alert-dialog-title'>{_('Resend fax?')}</DialogTitle>
```

File: web/portal/client/src/entities/FaxesOut/Action/Resend.tsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+        <DialogContent>
+          <DialogContentText id='alert-dialog-description'>
+            {_('You are about to resend this faxes')}
```
Comment (mmadariaga) [inline web/portal/client/src/entities/FaxesOut/Action/Resend.tsx:89]:
```suggestion
            {_('You are about to resend this fax')}
```

File: web/portal/client/src/entities/FaxesOut/Action/Resend.tsx [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <OutlinedButton onClick={handleClose}>Cancel</OutlinedButton>
```
Comment (mmadariaga) [inline web/portal/client/src/entities/FaxesOut/Action/Resend.tsx:93]: `Cancel` should be translatable

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

File: web/portal/client/src/translations/es.json [hunk 10]
```diff
@@ -456,6 +456,9 @@
   "Registered until: {{date}}": "Registrado hasta: {{date}}",
   "Regular Expression": "Expresión regular",
   "Regular NAT detected": "NAT normal detectado",
+  "Resend": "Reenviar",
+  "Resend fax": "Reenviar fax",
+  "Resend fax?": "Reenviar fax?",
```
Comment (Kaian) [inline web/portal/client/src/translations/es.json:461]: Missing ¿ symbol for spanish questions

**Review by Kaian — APPROVED** [review]

**Pull request merged**